### PR TITLE
fix: front settings fixes

### PR DESCRIPTION
**Pull request by JohnBerd**
- modify network settings screen to be a modal
- change arrow colors for settings accordions (darker grey)
- toggles work for Berty default nodes
- onboarding: "toggle all" toggles all node lists
- delete in app notification in settings (off-grid and network)
- restart app after leaving network settings if a node list, multi-peers, or BLE has changed
- restart app after leaving settings if "off grid" has changed
- if camera permissions are not granted, qr code is displayed by default in the "share view"
- ask camera permissions after clicking on toggle scan in the "share view", if not granted, scan is not toggled

### CODE REVIEW

**Comment by aeddi** [comment]
`restart app after leaving network settings if a node list, multi-peers, or BLE has changed`

The restart occurs when you leave the main settings page, not the network settings page.
The right behavior would be:
- when you edit something in network settings, if you leave the network settings screen (not main settings) -> restart
- when you edit the toggle in main settings, if you leave the main settings screen -> restart

**Comment by aeddi** [comment]
`onboarding: "toggle all" toggles all node lists`

It doesn't work properly. When I toggle off all, it works fine, but if I toggle on all, the `Multicast DNS` and `Distributed Hash Table` settings stay off.

**Comment by aeddi** [comment]
The `add menu` / `share menu` is still contained in a full page on iOS and not a form sheet.

**Comment by aeddi** [comment]
`modify network settings screen to be a modal`
It doesn't work properly. When I click on the `+` button of a nodelist to add something, the modal appears behind the form sheet.

File: js/packages/components/settings/Accounts.tsx [hunk 1]
```diff
@@ -10,7 +10,6 @@ import {
 	useThemeColor,
 	pbDateToNum,
 	closeAccountWithProgress,
-	exportAccountToFile,
```
Comment (glouvigny) [inline js/packages/components/settings/Accounts.tsx:13]: You should keep this, along `withInAppNotification` below

The backup feature should not be exposed on web

**Pull request merged**